trustpinning/certs_test.go

```diff
@@ -814,6 +814,52 @@ func generateRootKeyIDs(r *data.SignedRoot) {
 	}
 }
 
+func TestParsePEMPublicKey(t *testing.T) {
+	gun := "notary"
+	pass := func(keyName, alias string, createNew bool, attempts int) (passphrase string, giveup bool, err error) {
```

Comment:
Non-blocking:  I think you can use `passphrase.ConstantRetriever("password")` for this, unless you don't want the extra import.